docs/developer-docs/smart-contracts/maintain/canister-control.md

```diff
@@ -0,0 +1,75 @@
+# Control
+
+## Overview
+
+A canister has a basic permission model of a list of controllers which have administrative rights over the canister. A controller is specified by a principal, which can be self-authenticating, e.g. a `dfx` developer identity, or another canister. A canister can have multiple controllers, and each controller has the same administrative rights over the canister. A canister can also have no controller, in which case the canister becomes immutable and cannot be upgraded or deleted. 
+
+## Setting the controllers of a canister
+
+The controllers of a canister are set when the canister is created. If no explicit controller settings are specified, the default controller is the principal that created the canister. If the canister is created with `dfx` then the default controller list contains the developer identity and the principial of the associated cycles wallet canister.
+
```

Comment:
How would you define non-default controllers upon deployment?

Reply:
I don't t think this is used in practice a lot (except you do it programmatically from a canister) which I didn't want to discuss at this point.

Reply:
Ah, so you are saying when a canister creates another canister?

Reply:
Yes, exaclty.